webserver_test.py

```diff
@@ -81,13 +92,33 @@ def outputChecker(curl, curl_expected):
 sys.stdout.write(curl_output)
 sys.stdout.write("\n==========\n\n")
 if not curl_output == "<html><head><title>Not Found</title></head><body><h1>404 Not Found</h1></body></html>":
-    ec = 1
+    ec += 1
     sys.stdout.write("FAILED to match the following regular expression:\n")
     sys.stdout.write("  <html><head><title>Not Found</title></head><body><h1>404 Not Found</h1></body></html>\n")
+    
```

Comment:
Good job catching this; didn't notice
Thought it was being refactored in the previous pull request, not removed completely.